conf/machine/include/tune-arcv3.inc

```diff
@@ -0,0 +1,43 @@
+TUNE_ARCH = "${@bb.utils.contains('TUNE_FEATURES', 'hs58', 'arc32', 'arc64', d)}"
+
+TUNE_PKGARCH = "${TUNE_PKGARCH_tune-${DEFAULTTUNE}}"
+
+TUNEVALID[hs58] = "Enable ARCv3 64-bit"
+TUNE_CCARGS .= "${@bb.utils.contains('TUNE_FEATURES', 'hs58', ' -mcpu=hs5x', '', d)}"
+
+TUNEVALID[hs68] = "Enable ARCv3 64-bit core"
+TUNE_CCARGS .= "${@bb.utils.contains('TUNE_FEATURES', 'hs68', ' -mcpu=hs6x', '', d)}"
+
+TUNEVALID[ll64] = "Enable 64-bit load/store operations"
+TUNE_CCARGS .= "${@bb.utils.contains('TUNE_FEATURES', 'll64', ' -mll64', '', d)}"
+
+TUNEVALID[ll128] = "Enable 128-bit load/store operations"
+TUNE_CCARGS .= "${@bb.utils.contains('TUNE_FEATURES', 'll128', ' -m128', ' -mno-128', d)}"
+
+TUNEVALID[div_rem] = "Enable simple divider"
```

Comment:
Not sure really if we want to deal with so powerful new processors w/o HW division - it will compromise their purpose as a high-performance application processors then, no? :)